core/cautils/rootinfo.go

```diff
@@ -9,7 +9,7 @@ type RootInfo struct {
 
 	KSCloudBEURLs    string // Kubescape Cloud URL
 	KSCloudBEURLsDep string // Kubescape Cloud URL
-
+        KubeConfig       string //kubeconfig flag
```

Comment:
Not necessary.
Kubescape supports the `KUBECONFIG` env var

Reply:
Okay removed